src/Peering/Peering.Test/ScenarioTests/CreateNewDirectPeeringTests.ps1

```diff
@@ -0,0 +1,163 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+NewExchangeConnectionV4V6 
+#>
+function NewDirectConnectionV4V6($sessionPrefixv4,$sessionPrefixv6,$bandwidth)
+{
+    $resourceName = "testAkamaiEPV4V6"
+	$md5 = "25234523452123411fd234qdwfas3234"
+	$facilityId = "64"
+
+	$sessionv4 = "192.168.1." + $sessionPrefixv4
+	$sessionv6 = "fe01::" + $sessionPrefixv6
+	$maxv4 = 20000
+	$maxv6 = 2000
+
+    $resourceGroup = "testCarrier"
+    $resourceLocation = "CentralUS"
+    $profileSku = "Basic_Direct_Free"
+    $tags = @{"tag1" = "value1"; "tag2" = "value2"}
+    $createdConnection = New-AzPeeringDirectConnectionObject -PeeringDbFacilityId $facilityId -SessionPrefixV4 $sessionv4 -SessionPrefixV6 $sessionv6 -MaxPrefixesAdvertisedIPv4 $maxv4 -MaxPrefixesAdvertisedIPv6 $maxv6 -BandwidthInMbps $bandwidth -MD5AuthenticationKey $md5
+	return $createdConnection
+}
+
+<#
+.SYNOPSIS
+Tests new Direct Peering 
+#>
+function Test-NewDirectPeering
+{
+	$resourceName = "NewDirectPeering"
+    $resourceGroup = "testCarrier"
+    $peeringLocation = "Amsterdam"
+    $profileSku = "Basic_Direct_Free"
+	$asn = "/subscriptions/4445bf11-61c4-436f-a940-60194f8aca57/providers/Microsoft.Peering/peerAsns/Contoso"
```

Comment:
So i tried really hard to meet this requirement. Because it requires a lot of admin approvals to make a peering based on feature flag and PeerAsn approval.